linux_os/guide/system/accounts/accounts-restrictions/password_expiration/accounts_password_warn_age_login_defs/bash/shared.sh

```diff
@@ -1,9 +1,11 @@
-# platform = Red Hat Enterprise Linux 7,Red Hat Enterprise Linux 8,Red Hat Virtualization 4
+# platform = multi_platform_all
 . /usr/share/scap-security-guide/remediation_functions
+declare var_accounts_password_warn_age_login_defs
 {{{ bash_instantiate_variables("var_accounts_password_warn_age_login_defs") }}}
 
 grep -q ^PASS_WARN_AGE /etc/login.defs && \
-  sed -i "s/PASS_WARN_AGE.*/PASS_WARN_AGE     $var_accounts_password_warn_age_login_defs/g" /etc/login.defs
```

Comment:
Is this change intentional? It seems a bit out of scope of this PR.

Reply:
Some of the remediations were duplicated for different product, so this one is a fusion of multiple similar ones.